### PR TITLE
Update shello to work with recent git vello.

### PR DESCRIPTION
**Pull request by jneem**

### CODE REVIEW

**Comment by raphlinus** [comment]
In discussion with @dfrg yesterday, I think we decided we want to pin the SHA of git dependencies like vello, to avoid random breakage like we've just seen.

On a quick scan, the fix itself looks good, but I didn't read it critically.

**Comment by jneem** [comment]
Oh good point, I'll add the rev-pinning to this PR.

File: examples/shello.rs [hunk 1]
```diff
@@ -95,15 +96,13 @@ impl WindowState {
...
-                    &self.render.queue,
-                    &self.scene,
-                    &surface_texture,
-                    width,
-                    height,
-                )
+            let dev_id = surface.dev_id;
+            let device = &self.render.devices[dev_id].device;
+            let queue = &self.render.devices[dev_id].queue;
+            let mut renderer = Renderer::new(device).unwrap();
```
[the remark this answers is not in the document]
Comment (jneem) [reply examples/shello.rs]: Thanks! I pulled it in here.

**Review by raphlinus — APPROVED** [review]

**Comment by raphlinus** [comment]
Thanks! Looks good to me now.

**Pull request merged**